bsmetadata/preprocessing_utils.py

```diff
@@ -17,8 +17,16 @@
 from typing import Dict, List, Optional
 from urllib.parse import unquote, urlsplit
 
-from bsmetadata.vendor.dateutil.src.dateutil.parser import ParserError, parse
+import re
+from urllib.parse import urlparse
+import urllib
+
+import sys
+from bsmetadata import metadata_processors
+
+sys.path.insert(0, '/Users/christopher/git/metadata/')
```

Comment:
This line shouldn't be merged.

You can use it in your local copy, or you can try this in command line:
`export PYTHONPATH=$PYTHONPATH:$(pwd)`

Reply:
✅